website/content/docs/concepts/oidc-provider.mdx

```diff
@@ -0,0 +1,152 @@
+---
+layout: docs
+page_title: OIDC Provider
+description: >-
+  Describes how Vault can be an OIDC identity provider.
+---
+
+# OIDC Provider
+
+~> **Note**: The Vault OIDC Provider feature currently only supports the [authorization code flow](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
```

Comment:
I'm not sure that this detail needs an emphasized callout at the top of the document. I suggest that we incorporate this into a "Supported Flows" section or note it in sections covering OIDC-specific concepts.